src/server/db_slice.cc

```diff
@@ -503,13 +503,14 @@ DbSlice::ItAndExp DbSlice::FindInternal(const Context& cntx, std::string_view ke
   return res;
 }
 
-OpResult<pair<PrimeIterator, unsigned>> DbSlice::FindFirst(const Context& cntx, ArgSlice args,
-                                                           int req_obj_type) {
+OpResult<pair<PrimeConstIterator, unsigned>> DbSlice::FindFirstReadOnly(const Context& cntx,
+                                                                        ArgSlice args,
+                                                                        int req_obj_type) {
   DCHECK(!args.empty());
 
   for (unsigned i = 0; i < args.size(); ++i) {
     string_view s = args[i];
-    OpResult<PrimeIterator> res = Find(cntx, s, req_obj_type);
```

Comment:
I believe now Find is not used any more and can be removed right?

Reply:
Right! How could I forget :facepalm: